build-tools/packages/build-tools/src/fluidTsc/fluidTsc.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { defaultLogger } from "../common/logging";
+import { tsCompile } from "../common/tsCompile";
+
+const { log, errorLog: error } = defaultLogger;
+
+function printUsage() {
+	log(
+		`
+Usage: fluid-tsc [commonjs|module] [<tsc args>...]
+    [commonjs|module] value for package.json "type" property
+    [<tsc args>...] arguments passed to Typescript compiler (see tsc -?)
+`,
```

Comment:
Can we add a brief description of what it does or when it's supposed to be used, too?